services/base.js

```diff
@@ -82,23 +82,30 @@ class BaseService {
     return '/' + [this.url.base, partialUrl].filter(Boolean).join('/');
   }
 
+  static _makeStaticUrl(serviceData) {
```

Comment:
How about `_renderToStaticBadgeUrl`?

Reply:
It feels like this function and `_makeFullUrl()` are a pair of related functions and would benefit from similar names. If you're not a fan of `_makeStaticUrl()`, maybe `_makeExampleUrl()` or `_makeStaticExampleUrl()` would be better?

Reply:
`_makeStaticExampleUrl` works for me. There is also a function https://github.com/badges/shields/blob/66d444aa40703d401b5f13790eabd2259f136e3f/frontend/lib/badge-url.spec.js#L39-L43 which you might be able to use. (FWIW I'm not thrilled with the helpers in that module. Though I did write them, at last read they all confused me.)